pandas/_libs/tslibs/timezones.pyx

```diff
@@ -30,6 +32,10 @@ cdef int64_t NPY_NAT = get_nat()
 # ----------------------------------------------------------------------
 
 cdef inline bint is_utc(object tz):
+    if PY3:
```

Comment:
this would be a perf issue, you can almost certainly access this in stead at the cython level